test/lib/ufe/testComboCmd.py

```diff
@@ -395,3 +414,122 @@ def ball35Scale():
+        sn.append(usdSphereItem)
+
+        # Rotate both spheres around X, and scale them.
+        cmds.rotate(30, 0, 0, r=True, os=True, fo=True)
+        cmds.scale(1, 1, 2, r=True)
+
+        # Move pivots in world space.  At time of writing (20-Oct-20) UFE
+        # rotate pivot and scale pivot arguments to move command doesn't accept
+        # an object argument, so use the selection.
+        cmds.move(0, -2.104143, 3.139701, "pSphere1.scalePivot", "pSphere1.rotatePivot", r=True)
+        sn.remove(mayaSphereItem)
+        cmds.move(0, -2.104143, 3.139701, r=True, urp=True, usp=True)        
+
+        checkPivotsAndCompensations(self, "pSphere1", usdSphereT3d)
+
+        # Scale the spheres again
+        sn.append(mayaSphereItem)
+        cmds.scale(1, 1, 2, r=True)
+
+        # Move the pivots again.
+        cmds.move(0, 5.610465, 3.239203, "pSphere1.scalePivot", "pSphere1.rotatePivot", r=True)
+        sn.remove(mayaSphereItem)
+        cmds.move(0, 5.610465, 3.239203, r=True, urp=True, usp=True)  
+
+        checkPivotsAndCompensations(self, "pSphere1", usdSphereT3d)
+
+        # Move only the rotate pivot.
+        cmds.move(0, 0, 3, r=True, urp=True)
+        cmds.move(0, 0, 3, "pSphere1.rotatePivot", r=True)        
+
+        checkPivotsAndCompensations(self, "pSphere1", usdSphereT3d)
+
+        # Move only the scale pivot.
+        cmds.move(0, 0, -4, r=True, usp=True)
+        cmds.move(0, 0, -4, "pSphere1.scalePivot", r=True)
+
+        checkPivotsAndCompensations(self, "pSphere1", usdSphereT3d)
+
+    def testRotateScalePivotCompensationAfterExport(self):
+        '''Rotate and scale pivots must match after export.'''
+
+        cmds.file(new=True, force=True)
+        mayaSphere = cmds.polySphere()[0]
+        
+        cmds.rotate(0, 0, -45, r=True, os=True, fo=True)
+        cmds.scale(4, 3, 2, r=True)
+        cmds.move(-2, -3, -4, "pSphere1.rotatePivot", r=True)
+        cmds.move(7, 6, 5, "pSphere1.scalePivot", r=True)
+
+        # Export out, reference back in using proxy shape.
+        usdFilePath = os.path.abspath('UsdExportMayaXformStack.usda')
+        cmds.mayaUSDExport(file=usdFilePath)
+
+        # Reference it back in.
+        proxyShape = cmds.createNode('mayaUsdProxyShape')
+        cmds.setAttr('mayaUsdProxyShape1.filePath', usdFilePath, type='string')
+
+        # MAYA-101766: awkward plug access for non-interactive stage loading.
+        outStageData = nameToPlug('mayaUsdProxyShape1.outStageData')
+        outStageData.asMDataHandle()
+
+        proxyShapeMayaPath = cmds.ls(proxyShape, long=True)[0]
+        proxyShapePathSegment = mayaUtils.createUfePathSegment(
+            proxyShapeMayaPath)
+        
+        spherePathSegment = usdUtils.createUfePathSegment('/pSphere1')
+        spherePath = ufe.Path([proxyShapePathSegment, spherePathSegment])
+        sphereItem = ufe.Hierarchy.createItem(spherePath)
+        usdSphereT3d = ufe.Transform3d.transform3d(sphereItem)
```

Comment:
As I'm writing this I realize I need the same test as line 439, if the created Transform3d interface does not have rotate and scale pivot compensation capability...